docs/integrations/data-integrations/couchbase.mdx

```diff
@@ -20,14 +20,16 @@ Before proceeding, ensure the following prerequisites are met:
 This handler is implemented using the `couchbase` library, the Python driver for Couchbase.
 
 The required arguments to establish a connection are as follows:
-* `host`: the host name or IP address of the Couchbase server
+* `connection_string`: the Connection string to specify the cluster endpoint
```

Comment:
Let's make this,
the connection string for the endpoint of the Couchbase server

Please make the same change in the README.